2009-04-23  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/sys/mount.h: add MNT_DETACH and
	MNT_EXPIRE.

diff --git a/sysdeps/unix/sysv/linux/sys/mount.h b/sysdeps/unix/sysv/linux/sys/mount.h
index b305549..0e75dbf 100644
--- a/sysdeps/unix/sysv/linux/sys/mount.h
+++ b/sysdeps/unix/sysv/linux/sys/mount.h
@@ -96,8 +96,12 @@ enum
 /* Possible value for FLAGS parameter of `umount2'.  */
 enum
 {
-  MNT_FORCE = 1			/* Force unmounting.  */
+  MNT_FORCE = 1,		/* Force unmounting.  */
 #define MNT_FORCE MNT_FORCE
+  MNT_DETACH = 2,		/* Just detach from the tree */
+#define MNT_DETACH MNT_DETACH
+  MNT_EXPIRE = 4		/* Mark for expiry */
+#define MNT_EXPIRE MNT_EXPIRE
 };
 
 
